components/KiboAppHeader.vue

```diff
@@ -1,11 +1,66 @@
 <template>
   <div>
+    <SfIcon class="sf-header__icon" @click.prevent="showHamburgerManu = true">
+      <font-awesome-icon :icon="['fas', showHamburgerManu ? 'times' : 'bars']" class="fa-icon" />
+    </SfIcon>
+
+    <KiboHamburgerMenu
+      v-show="showHamburgerManu"
+      class="sf-sidebar--left"
+      :title="sideBarTitle"
+      :subtitle="sideBarSubTitle"
+      :heading-level="3"
+      button
+      overlay
+      @closeHamburgerMenu="showHamburgerManu = false"
+    >
+      <template #content-top>
+        <SfButton
+          class="sf-button--pure sf-header__action sf-header__bottom-link sf-header__account"
+          @click="handleAccountClick"
+        >
+          <SfIcon class="sf-header__icon">
+            <font-awesome-icon :icon="[accountIcon, 'user-circle']" class="fa-icon" />
+          </SfIcon>
+          <div class="icon-name-sidebar">
+            <span> My Account</span>
```

Comment:
Use the lang en.js file, don't hardcode `My Account`

Reply:
Resolved.